typescript/sdk/src/consts/agentStartBlocks.ts

```diff
@@ -0,0 +1,26 @@
+import { ChainMap } from '../types';
+
+// TODO this was previously in hyp-deploy, but ideally should be integrated
+// into the ChainMetadata type
+export const agentStartBlocks: ChainMap<number> = {
```

Comment:
not sure why we need this in the SDK

Reply:
Tbh just not sure where else it should live. It's needed for making agent configs that don't try to index too far back.